docs/content/en/docs/packages/metallb/addmetallb.md

```diff
@@ -13,6 +13,9 @@ Be sure to refer to the [troubleshooting guide]({{< relref "../troubleshoot" >}}
   {{% alert title="Important" color="warning" %}}
    * Starting at `eksctl anywhere` version `v0.12.0`, packages on workload clusters are remotely managed by the management cluster.
    * While following this guide to install packages on a workload cluster, please make sure the `kubeconfig` is pointing to the management cluster that was used to create the workload cluster. The only exception is the `kubectl create namespace` command below, which should be run with `kubeconfig` pointing to the workload cluster.
+   * Operators can also integrate with Isovalent's for 3rd party BGP support. Isovalent can integrate with:
```

Comment:
I'm not sure this is the right place for this. @chrisnegus do you have any thoughts?

Reply:
Good question. I've asked @csplinter to weigh in.

Reply:
Would you mind adding a note to the opening comment of each PR in this stack indicating which PR comes before it - it greatly helps with navigability, thanks.

@chrisdoherty4 sure. I will follow this for the next set of Pull Requests that I will raise. Also, when you @chrisnegus @csplinter can advice on where to add this could you kindly edit this line as

Operators can also integrate with Cilium for 3rd party BGP support. Cilium can integrate with:

Reply:
@csplinter yes it does augment/ integrate with MetalLB.